packages/wallets/exodus/src/index.ts

```diff
@@ -1,16 +1,37 @@
-import { WalletAdapterType } from '@reef-knot/core-react';
+import { WalletAdapterType } from '@reef-knot/types';
+import { Ethereum } from '@wagmi/core';
+import { InjectedConnector } from 'wagmi/connectors/injected';
 import WalletIcon from './icons/exodus.svg';
 
+declare global {
+  interface Window {
+    // @ts-expect-error Subsequent property declarations must have the same type
+    ethereum?: Ethereum & { isExodus?: boolean };
```

Comment:
to fix this error you should instead extend `interface Ethereum` with `isExodus`